server/src/main/scala/org/http4s/server/staticcontent/FileService.scala

```diff
@@ -56,7 +56,8 @@ object FileService {
       case Success(rootPath) =>
         TranslateUri(config.pathPrefix)(Kleisli {
           case request =>
-            request.pathInfo.split("/") match {
+            def resolvedPath: OptionT[F, Path] = request.pathInfo.split("/") match {
+              case Array() => OptionT.some(rootPath)
```

Comment:
This code is in the neighborhood of our recent CVE, so I'd like to make sure another maintainer is thinking about corner cases that may be introduced by this line.  But I think it is correct.

Reply:
This looks fine to me